docs/cheaha/migration_overview.md

```diff
@@ -0,0 +1,194 @@
+
+### GPFS 5 Compute Nodes
+
+Compute nodes are only able to run jobs from one of GPFS 4 or GPFS 5 so compute power has been split between the two storage systems. Due to hardware configuration constraints, some changes were made to the partitions on GPFS 5 that could potentially impact jobs.
+
+| Partition | Available Nodes | Notes |
+|---|---|---|
+| mainline | 20 (2560 cores) | Include AMD CPUs. See [below](#changes-to-mainline-partitions) for details |
+| pascalnodes | 0 | All pascalnodes will be moved during the 1st compute migration |
+| amperenodes | 5 (10 A100s) | 10 amperenodes will be added during the 1st compute migration with the remaining 5 added once the migration completes |
+| amperenodes-medium | 1 (2 A100s) | Nodes will be added to the amperenodes-medium partition during both compute migrations |
+| largemem | 0 | largemem and largmem-long nodes will remain on GPFS 4 until the full migration completes. If you require access to the 1.5 TiB RAM nodes, contact support |
+
+#### Changes to Mainline Partitions
+
+On GPFS 4, all nodes in the interactive/express/short/medium/long (mainline) partitions have Intel processors while the `amd-hdr100` nodes have AMD processors. On GPFS 5, the mainline partitions use nodes that were previously migrated from the `amd-hdr100` partition and so now use AMD CPUs.
+
+In rare cases, tools compiled on on Intel processor can cause `Illegal Instruction` errors when run on AMD CPUs. If your jobs were submitted to one of the mainline partitions and show this error after migration, please [contact support](../index.md#how-to-contact-us). As the migration continues, the standard Intel nodes will also be migrated from GPFS 4 to GPFS 5 and added to the mainline partitions.
+
+#### Compute Migration
+
+To best accommodate workloads on both clusters during data migration, compute nodes will be migrated in two stages: at 50% and 100% data migration. Due to hardware networking and rack constraints, we have limited options as far as which nodes we can move at which times. See below for a list of which nodes will be migrated to GPFS 5 at which time.
+
+**50% Migration Completion**:
+
+- All pascalnodes (18 total nodes on GPFS 5, 72 available P100s)
+- 10 amperenodes (15 total on GPFS 5, 30 available A100s)
+
+**100% Migration Completion**:
+
+- 5 amperenodes (20 total on GPFS 5)
+- All largemem nodes (14 total on GPFS 5)
+
+### Directory Paths
+
+We have reorganized our GPFS 5 filesets to improve file access performance specifically for our network scratch space. As part of this, we have changed the `/data` drive to `/gpfs` and have brought `/scratch` under it.
+
+**Path conversions from GPFS 4 to GPFS 5**:
```

Comment:
Please check with JP, my understanding is that we don't intend to expose the `/gpfs/` path to users.

Reply:
@jprorama Thoughts?

Reply:
We are now using bind mount not symlinks.  The intention is to stick with bind mounts as they are first class mounts. The kernel doesn't see a difference.

The point of sticking with /scratch and /data is that we could (at some future date) swap out the devices that provide these paths and not affect user jobs.  One practical example is bursting to the cloud.  Basically, but maintaining a stable file namespace we don't have to change jobs just because the storage system changes.

Reply:
I've completely removed the section on Directory Paths since we are planning to keep the GPFS 4 naming scheme for the long-term. We will need to keep in mind the potential issue I mentioned above for the future. I'm not sure if the directories were symlinks or bind mounts when Lindsay ran into the error I mentioned. We'll need to do some testing with it

Reply:
Ok that is perfectly fair! Glad we were able to get this resolved.